doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,189 @@
+"""
+=======================================
+Interactive Tools and Apps with HoloViz
+=======================================
+
+This tutorial shows how to build interactive tools and applications
+using `Panel <https://panel.holoviz.org>`_, `HoloViews <https://holoviews.org>`_ you can use to make you and your teams
+analysis more interactive and powerful.

```

Comment:
```suggestion
using `Panel <https://panel.holoviz.org>`_ and `HoloViews <https://holoviews.org>`_.
These tools can make your analyses more interactive and, hence, powerful.
```

Reply:
After this very general introduction, please introduce what the example application is about, so the reader knows where we're headed!